roles/tpa_single_node/vars/main.yml

```diff
@@ -18,46 +18,48 @@ tpa_single_node_pg_user: guac
 tpa_single_node_pg_user_passwd: guac1234
 tpa_single_node_pg_ssl_mode: disable
 
-# Storage Service (S3 or Minio)
+# Storage Service - S3 or Minio
 tpa_single_node_storage_endpoint: # <Enter minio storage endpoint, For AWS this field is not necessary>
-tpa_single_node_storage_region: # <AWS S3 Storage region> # For Minio just keep us-west-1
+tpa_single_node_storage_region: eu-west-1 # <AWS S3 Storage region> - For Minio just keep us-west-1
 tpa_single_node_storage_access_key: "{{ lookup('env', 'TPA_STORAGE_ACCESS_KEY') }}"
 tpa_single_node_storage_secret_key: "{{ lookup('env', 'TPA_STORAGE_SECRET_KEY') }}"
-tpa_single_node_storage_bombastic_bucket: bombastic-default # <bombastic storage bucket name>
-tpa_single_node_storage_v11y_bucket: v11y-default # <v11y storage bucket name>
+tpa_single_node_storage_bombastic_bucket: bombastic-rhtpa
```

Comment:
Shall we leave the S3 bucket and Events fields as blank and have these as examples?

Reply:
@mrrajan , @desmax74 , 

I'm not sure about that because a default value is easier to change with a search/replace and it gives example values to the user. And it's better than comments cluttering the code.